plugin/src/main/groovy/com/novoda/gradle/command/AndroidCommandPluginExtension.groovy

```diff
@@ -83,6 +86,14 @@ public class AndroidCommandPluginExtension {
         monkey
     }
 
+    void script(Action<NamedDomainObjectContainer<InputSpec>> action) {
```

Comment:
That line in BPP adds an extension to the project, right? I want to add the same to the `command` extension we already have.
Should I do something like `project.android.command.extensions.add()`?

And I didn't understand how it will help being lazy. All the code in the extension closure runs immediately with any gradle task, right?